lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -46,11 +46,20 @@
                          [alt]="'ADF_VIEWER.ARIA.MIME_TYPE_ICON' | translate"
                          [src]="mimeTypeIconUrl"
                          data-automation-id="adf-file-thumbnail">
-                    <div class="adf-viewer__display-name"
-                         id="adf-viewer-display-name"
-                         [title]="fileName">
-                        <span>{{ displayName }}</span>
-                    </div>
+                    @if (displayTitle) {
+                        <div class="adf-viewer__title-display-name-container"
+                             id="adf-viewer-title-display-name"
+                             [title]="displayTitle">
+                            <p class="adf-viewer__title-value">{{ displayTitle }}</p>
```

Comment:
Can we add a test case to check if both are displayed in case title is provided?

Reply:
of course I can, I was even thinking to test it, but finally I decided it would testing a framework @ng IF which obviously works if you pass a true value, there is no logic behind it. What do you think?